### PR TITLE
sig-node: run kubelet standalone tests periodically

### PR DESCRIPTION
**Pull request by tzneal**

### CODE REVIEW

File: config/jobs/kubernetes/sig-node/node-kubelet.yaml [hunk 1]
```diff
@@ -285,6 +285,89 @@ periodics:
             cpu: 4
             memory: 6Gi
 
+
+- name: pull-kubernetes-node-e2e-containerd-standalone-mode-all-alpha
```
Comment (dims) [inline config/jobs/kubernetes/sig-node/node-kubelet.yaml:289]: periodic jobs start with `ci-`

Comment (tzneal) [reply config/jobs/kubernetes/sig-node/node-kubelet.yaml]: Thanks!

**Comment by SergeyKanzhelev** [comment]
@tzneal tests are not running: 

- https://testgrid.k8s.io/sig-node-release-blocking#node-kubelet-containerd-standalone-mode-all-alpha
- https://testgrid.k8s.io/sig-node-release-blocking#node-kubelet-containerd-standalone-mode

Can you please take a look?

**Comment by SergeyKanzhelev** [comment]
Oh, I see the agenda item fro tomorrow meeting

**Pull request merged**